### DISCUSSION TITLE
I deployed pocketbase through dokploy templates, but there was no link for me to set up superuser, but I was directly allowed to log in.

### DISCUSSION THREAD

**Discussion by huglemon** [pocketbase/pocketbase#6036] [Q&A]
Where should I set or find this account and password?

**Comment by ganigeorgiev** [comment]
It will depend on what PocketBase version the template you are referring is using.

If it is using PocketBase v0.23+ then by default we create a system superuser account with random password and a link to setup it is printed in the console/stderr (I'm not familiar with Dokploy but try to look for the logs related to your deployment in their dashboard). Alternatively you can always create your superuser account manually by running `./pocketbase superuser create EMAIL PASS` (if you are using a custom pb_data you need to specify `--dir` too with the superuser command).

**Reply by ganigeorgiev** [reply]
Based on the template source in https://github.com/Dokploy/dokploy/blob/canary/apps/dokploy/templates/pocketbase/docker-compose.yml they are using [SPECTADO/pocketbase-docker](https://github.com/SPECTADO/pocketbase-docker) as Docker image, meaning that if you want to manually set a superuser password you'll have to ssh into the container and run something like:

```sh
/usr/local/bin/pocketbase superuser create EMAIL PASS --dir="/pb_data"
```

**Reply by huglemon** [reply]
I think I found the problem. Dokploy should not be adapted somewhere, which caused the url not to be printed in the logs after I deployed it.
Then the pocketbase command is not in the root directory, but in /usr/local/bin. At this time, you have to specify the directory as /pb_data
Thank you for your reply. I have created superuser and now I will go to dokploy to report this issue.
